packages/grap/build.sh

```diff
@@ -0,0 +1,16 @@
+TERMUX_PKG_HOMEPAGE=https://www.lunabase.org/~faber/Vault/software/grap/
+TERMUX_PKG_DESCRIPTION="Grap allow to display graphs in a Groff workflow."
+TERMUX_PKG_LICENSE="BSD"
```

Comment:
It looks like specifying the license file will be necessary here since `COPYING` just links to the README.
The license can also be clarified as BSD 2-Clause.
```suggestion
TERMUX_PKG_LICENSE="BSD 2-Clause"
TERMUX_LICENSE_FILE="COPYRIGHT"
```